### DISCUSSION TITLE
about-obsidian/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#16] [Announcements]
# about-obsidian/

تقریباً یک‌سالی میشه که از نرم‌افزار Obsidian استفاده می‌کنم و بسیار بسیار ازش راضیم.
از Obsidian برای گزارش‌نویسی روزانه (و هفتگی و ماهانه و فصلی و سالانه)، خلاصه‌نویسی، انواع نوت‌برداری‌ها و حتی مدیریت این وبلاگ و وبلاگ انگلیسی‌ام استفاده می‌کنم.
گفتم احتمال خوبه که در یک پست به معرفی‌اش بپردازم و ویژگی‌هاش رو بگم.
از مدت‌ها پیش که محمدرضا دائم توصیه به نوشتن می‌کرد، روش‌های مختلف نوشتن رو تست کردم. مدتی روی دفتر و کاغذ می‌نوشتم ولی زیاد حس خوبی نداشتم.

https://aprd.ir/about-obsidian/

**Comment by hiwasham** [comment]
سلام امیر جان
منم 4 سال و پنج ماهه از ابسیدین استفاده میکنم. ابزار فوق العاده قدرتمندیه. البته من با چند سیسم دیگه هم ترکیبش کردم و یه سری روتین و ورکفلو دارم برای هر کدوم. مثلا با 
GTD
Second Brain
Incremental Reading/Writing (SuperMemo)
Zettelkasten
تو رو هم فالو میکنم که از تجربیات و دانش خوبی که داری استفاده کنم
دمت شما گرم

**Reply by pourmand1376** [reply]
سلام هیوا جان.
چه عالی. 

راستش من زیاد تو یوتیوب دیدم که افراد مختلف روش‌های Zettelkasten و امثالهم رو آموزش میدن؛ ولی هیچ‌وقت نتونستم خودم انجام‌شون بدم. 

کاش شما هم اگر فرصت کردی بنویسی که چطوری از Obsidian استفاده می‌کنی که بتونیم ایده بگیریم. این Obsidian انقدر Customizable نوشته شده که هر کسی می‌تونه با یک روش کاملاً متفاوتی از بقیه ازش استفاده کنه و خیلی هم راضی باشه.

**Reply by hiwasham** [reply]
چند بار سعی کردم اینکارو بکنم اما دیدم شدنی نیست چون موضوع خیلی مفصلیه. کلی مقدمات داره. اول باید اون مدل ذهنی و لزوم نیاز به این ورکفلوها و سیستمها رو بگیم بعد برسیم به ابزار که حالا میتونه نوشن باشه یا ابسیدین یا هر ابزار دیگه ای
وگرنه طرف سردرگم و حتی زده میشه
خیلی وقته تو ذهنمه یه دوره براش بسازم ولی هنوز فرصتش رو پیدا نکردم. حتما اینکارو میکنم یه روزی
البته یه عده مثل خودمون با همین مقالات و ویدئوها و کتابها کارشون رو راه میندازن ولی برای عموم اگه یه دوره جامع تر باشه کار رو براشون راحت تر میشه

**Comment by zachshirow** [comment]
سلام آقا دست شما درد نکنه من رو با این giscus آشنا کردین. به وبسایتم اضافه اش کردم. ممنونم. 

یکی از پست هام که این سیستم کامنت رو دارند: 

https://zachshirow.ir/posts/obsidian-farsi-community

**Comment by hoseinmaddahi75** [comment]
درسته این مطلب خیلی وقت پیش نوشته شده ولی خب دوست داشتم تجربه خودم رو هم به اشتراک بذارم با بقیه.
در رابطه با سینک کردن نرم افزار توی پلتفرم های مختلف، من از دراپ باکس استفاده می کنم.
روال کار هم اینه که یادداشت های کامپیوتر رو توی دراپ باکس ذخیره می کنم. توی گوشی هم همینطور. یک نرم افزار به اسم dropsync هست که میتونه دراپ باکس رو سینک کنه. هر زمان وارد نرم افزار بشید می تونید به سادگی یادداشت ها رو سینک کنید بین گوشی و کامپیوتر.  یک کم دردسر داره اولش ولی خب کار که انجام بشه بعدش همه چیز راحته.

**Reply by pourmand1376** [reply]
سلام حسین جان. ممنون از اشتراک گذاری تجربه‌ات. 

یکی از دغدغه‌های اصلی هر کسی موقع نصب Obsidian همین سینک کردنش هست. من چند وقت اخیر با Gitsync کار رو انجام دادم و راضی بودم. بابت پیشنهادت هم ممنونم. در اولین فرصتی که ببینم داره اذیت می‌کنه میام سراغ راه‌حل تو ببینم چطوریه😅 فعلاً که خدا رو شکر اذیت نکرده. 

https://github.com/ViscousPot/GitSync

**Comment by alidalvand** [comment]
فک کنم وقتشه این پست رو آپدیتش کنی و تجربیات جدید تر تو بنویسی.
